types/unindent/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for unindent 2.0
+// Project: https://github.com/curvedmark/unindent
+// Definitions by: Alex Gorbatchev <https://github.com/alexgorbatchev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace unindent;
+export = unindent;
+
+declare namespace unindent {}
```

Comment:
It seems this line can be removed

Reply:
This is required to get ES6 import to work `import * as unindentES6 from 'unindent';`

Reply:
It's a hack and discouraged: https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
I would really prefer to merge typings that allow myself and others to follow commonly established TS patterns. There are currently ~130 modules in typings that use this pattern, using alternative import syntax ins't an option for a large existing project (nor is there a reason to), all of the modules that export a single function and have typings are importable via `import * as foo...` syntax. If this module isn't it will trip everyone up.

Reply:
ping

Reply:
We don't recommend that pattern now that [`--esModuleInterop` ](https://www.typescriptlang.org/docs/handbook/compiler-options.html) is implemented. You should be able to use a default import `import unindent from "unindent";` if you need to use ES6 syntax.

Reply:
Thanks for your feedback. I'll close the PR and publish this on my own.